pkg/dpll/dpll.go

```diff
@@ -0,0 +1,268 @@
+package dpll
+
+import (
+	"encoding/binary"
+	"fmt"
+	"math"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift/linuxptp-daemon/pkg/config"
+	"github.com/openshift/linuxptp-daemon/pkg/event"
+)
+
+const (
+	DPLL_UNKNOW        = -1
+	DPLL_INVALID       = 0
+	DPLL_FREERUN       = 1
+	DPLL_LOCKED        = 2
+	DPLL_LOCKED_HO_ACQ = 3
+	DPLL_HOLOVER       = 4
```

Comment:
When we compare phase and frequency states we choose the lower one, which imply the later the number, the worse the signal.  Based on this the values should be the following:
```
	DPLL_HOLOVER       = 2
	DPLL_LOCKED_HO_ACQ = 3
	DPLL_LOCKED        = 4
```